src/scss/variables-admin.scss

```diff
@@ -43,6 +43,7 @@ $cadillac: #884488;
 //** Fonts variables
 $frutiger: 'Frutiger Neue', Helvetica, Arial, sans-serif;
 $admin-icons: 'Admin Icons';
+$admin-default-font-size: 0.825rem;
```

Comment:
as said in my last comment, I think this should go away.
https://github.com/geoadmin/web-mapviewer/pull/368#issuecomment-1437934824